biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/implicits/Either.scala

```diff
@@ -0,0 +1,40 @@
+package nl.lumc.sasc.biopet.utils.implicits
+
+/**
+ * Created by pjvanthof on 20/02/2017.
+ */
+object Either {
```

Comment:
If I recall correctly, many of the implicits here are already defined in the scalaz library (using a slightly different type, but scalaz is already a dependency of ours anyway). Plus, the mapping convention is to use `Left` for error messages and `Right` for the results. Are there cases where we map `Option` to `Left` in our codebase?